### DISCUSSION TITLE
Use STL file as a dump

### DISCUSSION THREAD

**Discussion by samerps** [thliebig/openEMS-Project#47] [Q&A Usage] [answered]
Is it possible to import an STL file and use that as a dump? I tried doing that but it didn't work, the output vtk files were empty. I'm simulating a geometry with several curvatures, I do not want to use a large rectangular dump box because the output files will be very large. Thanks

**Comment by thliebig** [comment, marked as answer]
No that is not possible. Any other object than a box should result in the bounding box used. But I have not tested that.
But there are plenty of ways to reduce data size.
E.g. dumping a specif frequency instead of very many time-domain samples. Using a sub-sampling or a target sampling if the simulation resolution is much finer than you need to be saved...
